src/app/components/ThemeProvider/withThemeProvider.tsx

```diff
@@ -4,6 +4,11 @@ import {
   ThemeProvider as EmotionThemeProvider,
   Theme,
 } from '@emotion/react';
+import {
+  ThemeComponent,
+  applyDarkPalette,
+  ThemeContextPOC,
+} from '#app/contexts/ThemeContextPOC';
```

Comment:
Just a note that on components following our coding standards we should avoid using aliases and use paths instead as on this line:
`import { RequestContext } from '../../contexts/RequestContext';` 

https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/React-Components.stories.mdx#typescript-for-frontend-components